src/ImageSharp/IDeepCloneable.cs

```diff
@@ -14,7 +14,7 @@ public interface IDeepCloneable<T>
         /// Creates a new <typeparamref name="T"/> that is a deep copy of the current instance.
         /// </summary>
         /// <returns>The <typeparamref name="T"/>.</returns>
-        T DeepClone();
+        T Clone();
```

Comment:
I think more a verbose name is better in this specific case. What if an object is both deep an shallow cloneable? (Like `Configuration`)

Reply:
It just looked weird that's all when we had a mix of both `Clone()` and `DeepClone()` together. I'll revert it though.